.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,42 @@
+name: Feature Request
+description: Request a new feature
+title: "[FEAT]: "
+labels: ["enhancement"]
+assignees:
+  - heubeck
+  - johannesmarx
+  - beiertu-mms
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### Thank you for taking your time to fill out this form!
+        Please also search the [issues](https://github.com/MediaMarktSaturn/technolinator/issues) first before submitting a new one.
+
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected Behavior
+      description: |
+        Tell us what should be improved/added/changed/removed.
+    validations:
+      required: true
+
+  - type: textarea
```

Comment:
does this make sense for feature requests?

Reply:
not sure, but maybe these two text areas can be combined into one called e.g. Request Description?

Reply:
these are now merged into one.